### PR TITLE
VP-3645 x order

### PR DESCRIPTION
**Pull request by kostyrin**

### CODE REVIEW

File: src/XPurchase/VirtoCommerce.ExperienceApiModule.XOrder/Queries/GetOrderQueryHandler.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    public class GetOrderQueryHandler : IQueryHandler<GetOrderQuery, CustomerOrder>
+    {
+        private readonly ICustomerOrderAggregateRepository _customerOrderAggregateRepository;
+        public GetOrderQueryHandler(ICustomerOrderAggregateRepository customerOrderAggregateRepository)
+        {
+            _customerOrderAggregateRepository = customerOrderAggregateRepository;
+        }
+
+        public Task<CustomerOrder> Handle(GetOrderQuery request, CancellationToken cancellationToken)
+        {
```
Comment (eMazeika) [inline src/XPurchase/VirtoCommerce.ExperienceApiModule.XOrder/Queries/GetOrderQueryHandler.cs:18]: put validator call here

File: src/XPurchase/VirtoCommerce.ExperienceApiModule.XOrder/Queries/SearchOrderResponse.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+using VirtoCommerce.OrdersModule.Core.Model;
+
+namespace VirtoCommerce.ExperienceApiModule.XOrder.Queries
+{
+    public class SearchOrderResponse
+    {
+        public int TotalCount { get; set; }
+        public IList<CustomerOrder> Results { get; set; }
+        public IList<CustomerOrder> CustomerOrders => Results;
```
Comment (eMazeika) [inline src/XPurchase/VirtoCommerce.ExperienceApiModule.XOrder/Queries/SearchOrderResponse.cs:10]: remove the duplicating prop. Add field alias in query, if needed

File: src/XPurchase/VirtoCommerce.ExperienceApiModule.XOrder/Schemas/OrdersConnection.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+using GraphQL.Types.Relay.DataObjects;
+
+namespace VirtoCommerce.ExperienceApiModule.XOrder.Schemas
+{
+    public class OrdersConnection<TNode> : Connection<TNode>
```
Comment (eMazeika) [inline src/XPurchase/VirtoCommerce.ExperienceApiModule.XOrder/Schemas/OrdersConnection.cs:5]: remove, if class unused

File: src/XPurchase/VirtoCommerce.ExperienceApiModule.XOrder/Queries/GetOrderQueryHandler.cs (hunk 1 above)
Comment (kostyrin) [reply src/XPurchase/VirtoCommerce.ExperienceApiModule.XOrder/Queries/GetOrderQueryHandler.cs:18]: why do we need to use the validator for queries?

File: src/XPurchase/VirtoCommerce.ExperienceApiModule.XOrder/Schemas/OrdersConnection.cs (hunk 3 above)
Comment (kostyrin) [reply src/XPurchase/VirtoCommerce.ExperienceApiModule.XOrder/Schemas/OrdersConnection.cs]: done.

**Pull request merged**